docs/sources/setup-grafana/configure-security/configure-authentication/enhanced-ldap/index.md

```diff
@@ -18,7 +18,7 @@ weight: 900
 
 The enhanced LDAP integration adds additional functionality on top of the [LDAP integration]({{< relref "ldap/" >}}) available in the open source edition of Grafana.
 
-> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud).
+> **Note:** Available in [Grafana Enterprise]({{< relref "../../../../introduction/grafana-enterprise/" >}}) and [Grafana Cloud Advanced](/docs/grafana-cloud). If you are a Grafana Cloud customer, please contact Support to configure this feature.
```

Comment:
How do users contact support? Is there a form they complete, or do they need to call? If there's a form, we could add a link to it here.

Reply:
Sorry for the delay @chri2547! Customers contact Support by opening a ticket from their Cloud Portal. We can use:

If you are a Grafana Cloud customer, please [open a support ticket in the Cloud Portal](/profile/org#support) to request this feature.